src/state/lists/hooks.ts

```diff
@@ -136,14 +136,11 @@ function useCombinedTokenMapFromUrls(urls: string[] | undefined): TokenAddressMa
 }
 
 // filter out unsupported lists
-export function useActiveListUrls(includeDefaultLists = false): string[] | undefined {
-  const defaultList = includeDefaultLists
-    ? DEFAULT_LIST_OF_LISTS.filter((url) => !UNSUPPORTED_LIST_URLS.includes(url))
-    : []
+export function useActiveListUrls(): string[] | undefined {
```

Comment:
How about creating a list selector file `list/selectors.ts` ?

```
  const activeListUrlsSelector = (state) => state.lists.activeListUrls
  
  const combinedActiveListUrlsSelector = createSelector(activeListUrlsSelector, (urls) => urls.filter(...))
```

In `hooks.ts`: 
```

useActiveListUrls() { return useSelector(combinedActiveListUrlsSelector))
```